Dockerfile

```diff
@@ -1,24 +1,15 @@
 # Read the Docs - Environment base
-FROM ubuntu:16.04
+FROM readthedocs/build:base
 MAINTAINER Read the Docs <support@readthedocs.com>
 LABEL version="latest"
 
-ENV DEBIAN_FRONTEND noninteractive
-ENV APPDIR /app
-ENV LANG C.UTF-8
-
-# System dependencies
-RUN apt-get -y update
-RUN apt-get -y install vim software-properties-common
-
 # Install requirements
-RUN apt-get -y install \
-    bzr subversion git-core mercurial libpq-dev libxml2-dev libxslt-dev \
+RUN apt-get -y install libpq-dev libxml2-dev libxslt-dev \
```

Comment:
I think this will be a problem since we are already as a `docs` user from the base image :)